### ISSUE TITLE
Activities: ORDER_THANKYOU should be called ORDER_CONFIRMED

### ISSUE THREAD

**Issue by Betree** [opencollective/opencollective#5901]
We recently introduced an `ORDER_THANKYOU` activity. It is triggering the "Thank you" email sent to financial contributors.

I believe the name for this activity is not ideal; it shouldn't be based on the email template name (the action triggered) but rather on the root event (what happened originally). Looking at the code, the activity is generated when an order gets confirmed (aka. the payment gets accepted).

I therefore propose to rename `ORDER_THANKYOU` -> `ORDER_CONFIRMED`.

cc @kewitz

**Comment by kewitz** [comment]
Agreed. Should we update the value and migrate past activities as well?

**Comment by Betree** [comment]
I think so, the only regression I can think of is people who have a webhook for "All events" and are filtering on the type. But given we've introduced this one very recently, I don't think it will have a strong impact. I'm still going to announce this on https://opencollective.com/engineering/updates, just in case.

**Comment by znarf** [comment]
@Betree Thank you, good practice to announce breaking changes! Someone I know told me about the notification and found it nice.

**Comment by Betree** [comment]
@znarf Great to hear!

**Comment by Betree** [comment]
Note that, at the moment, this activity is virtual - we don't store it in the database. We're also missing an activity for "Order created".

**Comment by Betree** [comment]
Completed in https://github.com/opencollective/opencollective-api/pull/9688
